src/Template/Layout/default.ctp

```diff
@@ -15,38 +9,20 @@ echo $this->fetch(
             <?= $this->fetch('title', read('App.title', env('HTTP_HOST'))) ?>
         </title>
 
+        <meta name="viewport" content="width=device-width, initial-scale=1">
         <meta name="author" content="<?= read('App.author') ?>">
+        <?= $this->Html->meta('icon') ?>
         <?= $this->fetch('meta') ?>
-
-
         <?= $this->AssetCompress->css('platform') ?>
-        <?= $this->fetch('css'); ?>
-        <!-- Le HTML5 shim, for IE6-8 support of HTML5 elements -->
-        <!--[if lt IE 9]>
-        <script src="//html5shim.googlecode.com/svn/trunk/html5.js"></script>
-        <script src="//oss.maxcdn.com/libs/respond.js/1.3.0/respond.min.js"></script>
-        <![endif]-->
+        <?= $this->fetch('css') ?>
+        <?= $this->fetch('headjs') ?>
```

Comment:
I did not work with CRUD since CakePHP2 / CRUDv3 (custom build crud view). So bear with me if this comment has more the form of an RFC. I don't know what `<?= $this->AssetCompress->css('platform') ?>` does, if it includes below ignore:

I suggest adding the following:

1. core-application-top.css
2. core-application-top.js
3. some-plugin-application-top.css
4. some-plugin-application-top.js
5. core/pluginname-controllername-actionname-top.css
6. core/pluginname-controllername-acitonname-top.js
7. core-application-bottom.css
8. core-application-bottom.js
9. some-plugin-application-bottom.css
10. some-plugin-application-bottom.js
11. core/pluginname-controllername-actionname-bottom.css
12. core/pluginname-controllername-actionname-bottom.js

E.g.
1. for core css and js application top, and bottom are being loaded, same for each plugin (if called controller is part of that plugin).
2. each action (prefixed by controller and plugin (or nothing for core?) gets its css and js loaded
No statements if files don't exist and/or plugins and/or controllers and/or actions are not part of the request

Usually you should by doing so go well with one application-top.css, one application-bottom.css, one application-bottom.js, one action-bottom.css and one action-bottom.js. 5 requests but those allow dereferencing and decoupling. The application-* files will be in get cache anyway and thus only action-bottom.css and action-bottom.js will need to be loaded (2 requests after the first full page load). One could opt-in for eager loading by using prefixes (body class="pluginname-controllername-actionname" is the easiest) and just application-top.css and application-bottom.js.